### PR TITLE
Vars from secrets

### PR DESCRIPTION
**Pull request by crochcunill**
I have updated the github workflows to use variables (vars) instead of secrets. Also updated the Redme file.

Using the vars_from_secrets  branch I have been able to successfully destroy and then redeploy the serverless sample app. The new url is https://dhxdodynx1syg.cloudfront.net/ and one of the greetings message has been altered to show it is using vars instead of secrets. Secrets have been removed from the Settings and replaced by variables

### CODE REVIEW

File: client/src/components/form/Greeter.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ console.log(values)
             options={[
               { value: 'Aloha', label: 'Aloha' },
               { value: 'Bonjour', label: 'Bonjour' },
-              { value: 'Greetings and salutations', label: 'Greetings and salutations' },
+              { value: 'Greetings and salutations', label: 'o20ya1 - after changing secrets to vars - April 19, 2023' },
```
Comment (wrnu) [inline client/src/components/form/Greeter.js]: Let's undo this change after now that you have tested since this is the generic sample app that everyone clones and isn't tied to a specific license plate

**Review by wrnu — CHANGES_REQUESTED** [review]
Looks good, just the one comment

File: client/src/components/form/Greeter.js (hunk 1 above)
Comment (crochcunill) [reply client/src/components/form/Greeter.js]: Completely agree, this was an oversight on my side. I have just changed it.

**Review by wrnu — APPROVED** [review]
